src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -2,10 +2,23 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
 
+@Entity
+@Table(name = "movies")
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String title;
+
+    @OneToMany(fetch = FetchType.EAGER)
```

Comment:
```suggestion
    @ManyToMany(fetch = FetchType.EAGER)
```
не тільки багато акторів в одному фільмі але також і один актор в багатьох фільмах

Reply:
согласен, забаранил